Cargo.toml

```diff
@@ -21,20 +21,19 @@ name = "e2e-tests"
 path = "e2e_tests/main.rs"
 
 [dependencies]
+tokio = { version = "0.2", features = ["macros"] } # Only used in bin/e2e, not used in library.
 chrono = { version = "0.4", features = ["serde"] }
-clap = "2.33"
+clap = "2.33" # Only used in binaries, not used in library.
```

Comment:
This makes me realize that we may be packaging this wrong

Reply:
We should probably use [cargo workspaces](https://doc.rust-lang.org/book/ch14-03-cargo-workspaces.html) instead

Reply:
should we make an issue?

Reply:
https://github.com/cloudflare/cloudflare-rs/issues/77